scripts/developer_tools/lib/github_repo.py

```
@@ -0,0 +1,24 @@
+
+from __future__ import annotations
+
+import re
+import subprocess
+
+
+def get_repo_info() -> tuple[str, str]:
+    """Extract the GitHub owner and repository name from ``origin``."""
+    try:
+        result = subprocess.run(
+            ["git", "config", "--get", "remote.origin.url"],
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)